stacks/triliovault-operator/deploy.sh

```diff
@@ -13,7 +13,7 @@ helm repo update > /dev/null
 ################################################################################
 STACK="triliovault-operator"
 CHART="triliovault-operator/k8s-triliovault-operator"
-CHART_VERSION="2.6.7"
+CHART_VERSION="$(curl -s https://charts.k8strilio.net/trilio-stable/k8s-triliovault-operator/index.yaml | grep -m 1 appVersion | awk -F ':' '{gsub(/ /,""); print $2 }')"
```

Comment:
Removing the chart version altogether should install the latest version.

Reply:
yes actually, we need a version only for an upgrade, added `latest`

Can you help me understand how the upgrade works on DO? how DO will come to know that an application upgrade is available?

Reply:
@keladhruv resolved issues